core/util/notifications.py

```diff
@@ -21,6 +22,14 @@ class PushNotifications:
 
     VALID_TOKEN_TYPES = [DeviceTokenTypes.FCM_ANDROID, DeviceTokenTypes.FCM_IOS]
 
+    @classmethod
+    def notifiable_tokens(cls, patron: Patron):
+        return [
+            token
```

Comment:
This syntax is slick - I'm not familiar with it.

Reply:
list comprehensions FTW!